resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -60,6 +61,7 @@
     private boolean writableStackTraceEnabled = DEFAULT_WRITABLE_STACK_TRACE_ENABLED;
     private Duration waitDurationInOpenState = Duration.ofSeconds(DEFAULT_WAIT_DURATION_IN_OPEN_STATE);
```

Comment:
We could remove `waitDurationInOpenState` and replace it by using a constant default `waitIntervalFunctionInOpenState`

Reply:
That would be a breaking change to remove `getWaitDurationInOpenState()` from this class

Reply:
I can do that if you want, up to you

Reply:
No, its fine. I can adjust it.